install/empty_data.php

```diff
@@ -3451,6 +3472,21 @@ public function getEmptyData(): array
                 'notifications_id'         => '76',
                 'mode'                     => 'mailing',
                 'notificationtemplates_id' => 31,
+            ], [
+                'id'                       => 77,
+                'notifications_id'         => '77',
+                'mode'                     => 'mailing',
+                'notificationtemplates_id' => 4,
+            ], [
+                'id'                       => 78,
+                'notifications_id'         => '78',
+                'mode'                     => 'mailing',
+                'notificationtemplates_id' => 4,
+            ], [
+                'id'                       => 79,
+                'notifications_id'         => '79',
+                'mode'                     => 'mailing',
+                'notificationtemplates_id' => 4,
```

Comment:
`Problems` template id is `17`.
```suggestion
                'notificationtemplates_id' => 17,
```